packages/contracts/contracts/L1/L2OutputOracle.sol

```diff
@@ -0,0 +1,113 @@
+//SPDX-License-Identifier: MIT
```

Comment:
I tried very hard to make this appear as a diff against the MockL2OutputOracle.sol file which this was renamed from, but was not able to.